.github/workflows/build.yml

```diff
@@ -0,0 +1,41 @@
+name: Actions 😎
+
+on: [push, pull_request]
```

Comment:
Please use [workflow_dispatch ](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatch) instead